### DISCUSSION TITLE
docker后端部署

### DISCUSSION THREAD

**Discussion by DZDZzzZ** [metatube-community/jellyfin-plugin-metatube#325] [Q&A]
我按照教程通过“docker run -d -p 8080:8080 -v $PWD/config:/config --name metatube metatube/metatube-server:latest -dsn /config/metatube.db”部署后端，之后该怎么办，怎么在MetaTube插件（jellyfin）中找到对应的server url 和 Token

**Comment by dealex63** [comment]
Server url: 是在你運行docker的ip 
token 是在config裡面

**Reply by poko11** [reply]
我连不上本机IP。http://localhost:7070/不行，192.X.X.X:7070也不行

**Comment by Jang-Ha-ri** [comment]
问下如何才算部署成功了？在docker中多了个“container metatube”容器？
